### DISCUSSION TITLE
Montage tile size defined by montage's result?

### DISCUSSION THREAD

**Discussion by whugemann** [ImageMagick/ImageMagick#3755] [Help]
I am working on a Windows batch file that creates photo index prints of folders. I know, the basic command lines is something along the lines of

magick montage *.jpg -geometry 300x200+3+3 -tile 4x7 indexprint.jpg

The dimensions of the resulting index print is a function of the tile dimensions an the ordering of the images.
Is there, however, a way of defining the tile size the other way round, i.e. by the desired dimensions of the index print?
I would like to fill a DIN A4 page (297 x 210 mm) with a reasonable density, say 100 dpmm (254 dpi), which would give 2970 x 2100 pixel.

The tile layout will depend on the number of images in the folder, fewer images would be displayed larger. (There is the equivalent of the case statement somewhere in the batch file.)

Can I somehow compute the tile dimensions within the IM command line? This was impossible in IM Version 6 AFAIK. Anthony's Usage pages therefor give no hint on the solution of this problem.

**Comment by fmw42** [comment]
Take the desired output image width and height and divide by the number of tiles in each direction. That would give you the geometry WxH, if you use the geometry offsets (spacing) of zero.
